blocksuite/affine/blocks/root/src/edgeless/clipboard/clipboard.ts

```diff
@@ -126,6 +126,33 @@ export class EdgelessClipboardController extends PageClipboard {
       return;
     }
 
+    // Only when an image is selected, it can be pasted normally to page mode.
+    if (elements.length === 1 && isImageBlock(elements[0])) {
+      const element = elements[0];
+      const sourceId = element.props.sourceId$.peek();
+      if (!sourceId) return;
+
+      await this.std.clipboard.writeToClipboard(async items => {
+        const job = this.std.store.getTransformer();
+        await job.assetsManager.readFromBlob(sourceId);
+
+        const blob = job.assetsManager.getAssets().get(sourceId);
+        if (!blob) {
+          return items;
+        }
+
+        // TODO(@fundon): need improve here
+        const type = 'image/png';
```

Comment:
Maybe you can refer to this part of the code.
https://github.com/toeverything/AFFiNE/blob/8726b0e462ccd3ff3b6f8bc213700d795cd95da2/blocksuite/framework/store/src/transformer/assets.ts#L82-L91